### PR TITLE
[stable/kube-ops-view] Add ingress support & enhance service template metadata

### PR DESCRIPTION
**Pull request by tyrannasaurusbanks**
* Add optional ingress for kube-ops-view 
* Add optional metadata items (annotations + labels) to the service template

```release-note
Add optional ingress for kube-ops-view
```

### CODE REVIEW

**Comment by tyrannasaurusbanks** [comment]
CLA...?

**Comment by tyrannasaurusbanks** [comment]
💯 ?

**Comment by thelinuxfoundation** [comment]
bump for cla check

**Comment by hjacobs** [comment]
@tyrannasaurusbanks thanks for the PR! If you are using [kube-ops-view](https://github.com/hjacobs/kube-ops-view), I would like to hear feedback and improvement suggestions :smile:

**Comment by tyrannasaurusbanks** [comment]
I've just deployed it to one of our clusters as I thought it provided a pretty cool dashboard for our cluster footprint out-of-the-box, and trying to get it to pull in state from our other clusters is also highlighting some bumps we currently have with our auth-provisioning mechanism.
So thanks @hjacobs !

This PR is just me feeding back some functionality I added to work nicely with out internal-kluster setup: service annotations to allow us to create some ELB & an ingress to give us, well, an ingress!.
I've tried to make my additions optional, but i don't know how you feel about the potentially empty ingress.yaml?
Apologies if I've missed some helm/community conventions - this is my first contribution; is there anything you'd like me to add?

**Comment by hjacobs** [comment]
@tyrannasaurusbanks thanks for the feedback! I'm also not a helm expert myself (this was my first chart)..

**Comment by tyrannasaurusbanks** [comment]
@hjacobs or @linki would you be able to help me get a label 'needs-ok-to-test' added to this PR?

**Comment by hjacobs** [comment]
@tyrannasaurusbanks I don't have such super powers :smirk:, but I will ping @linki ..

File: stable/kube-ops-view/templates/ingress.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
```
Comment (prydonius) [inline stable/kube-ops-view/templates/ingress.yaml]: the `---` is not needed, Helm already separates manifests correctly

**Review by prydonius — CHANGES_REQUESTED** [review]
The values you've created should also be defined in the values.yaml file.

Also it would be great to update this to follow the same ingress pattern in #813.

**Comment by tyrannasaurusbanks** [comment]
@prydonius Thanks for the review! I've refactored the ingress to match the conventions you linked to and added all new values to values.yaml

File: stable/kube-ops-view/templates/ingress.yaml (hunk 1 above)
Comment (tyrannasaurusbanks) [reply stable/kube-ops-view/templates/ingress.yaml]: removed, thanks

**Comment by tyrannasaurusbanks** [comment]
are you happy with these changes @prydonius?

**Comment by seanknox** [comment]
@k8s-bot ok to test

**Comment by tyrannasaurusbanks** [comment]
@seanknox is there anything I need to do to trigger a test run or will the k8-bot pick it up in due-course?

**Review by prydonius — APPROVED** [review]

**Comment by prydonius** [comment]
@tyrannasaurusbanks The k8s bot will automatically run the tests, and they're passing so no worries there. This all lgtm, thanks for the contribution!

**Comment by tyrannasaurusbanks** [comment]
Thanks for the review @prydonius and for the help getting my first commit in!

**Pull request merged**